traits/category.py

```diff
@@ -88,6 +92,8 @@ def add_traits_meta_data ( self, bases, class_dict, base_traits,
 #  'Category' class:
 #-------------------------------------------------------------------------------
 
+
+@six.add_metaclass(MetaCategory)
```

Comment:
Personally, I prefer the decorator variant as I find it easier to read. 

That being said, here is a gotcha when using the decorator:

```python
import six

class Meta1(type):
    pass

class Meta2(type):
    pass

class CombinedMeta(Meta1, Meta2):
    pass

@six.add_metaclass(Meta1)
class Base1(object):
    pass

@six.add_metaclass(Meta2)
class Base2(object):
    pass

# This works as intended
class DerivedClass(six.with_metaclass(CombinedMeta, Base1, Base2)):
    pass

# This fails with a TypeError(metaclass conflict)
@six.add_metaclass(CombinedMeta)
class DerivedClass(Base1, Base2):
    pass
```
I found this while running your test suite(test_abc.py). As far as I am aware, there is no impact once the class is created.

In the six documentation there is nothing that mentions anything like that.